static/js/util/profile_edit.js

```diff
@@ -442,16 +442,17 @@ export function boundDateField(keySet, label, omitDay) {
  * Saves the profile and returns a promise, taking an optional function
  * to retrieve keys for validation of nested fields (e.g. profile.work_history)
```

Comment:
this docstring is out of date (refers to earlier, when the validation setup was more naive and we were merely checking for field presence).